README.md

```diff
@@ -108,6 +108,19 @@ cd react
 ../env.sh local npm start
 ```
 
+## Run Caching Locally
+
+To run caching locally, you will need to setup port forwarding from you local machine to GCP redis instance.
+1. gcloud auth login
+2. gcloud compute ssh redis-relay --zone=us-central1-a -- -N -L 6379:10.251.35.179:6379
+
+You should now be authenticated and have the relay running. Use redis-cli ping to check.
+
+3. If you have REDISHOST in your .env, comment it out. We want flask to point to localhost.
+4. Run flask server
```

Comment:
do you think we could automate this in `flask/run.sh`?
that script is only run locally

Reply:
hm possibly. let me look into that, i have limited exposure to bash scripting

Reply:
don't worry about it then, I can do it

Reply:
@lukemun pushed to this branch
now only need to run `deploy.sh --env=local react flask`

https://github.com/sentry-demos/empower/pull/621/commits/1a7e5043cfc80b64f3ed65c5b274d0d53e3dd527